commit 6af0bd69074725c8d8b10f07ed4ccc013d010322
Author: Faustin Lammler <faustin@fala.red>
Date:   Fri Jan 11 18:22:44 2019 -0300

    MDEV-15526 systemd unit files naming and installation
    
    Drop Aliases from the service file directive. Aliases in the service
    file only take effect when the service is enabled. This is the case
    because Aliases in service files do not have to be unique across various
    services.
    
    Shipping symlinks guarantees that one can always enable mariadb service
    using `systemctl enable mysql` or `systemctl enable mysqld` and makes
    the commands indempotent.

diff --git a/support-files/mariadb.service.in b/support-files/mariadb.service.in
index dca7a4c13a7..e61e35c52ee 100644
--- a/support-files/mariadb.service.in
+++ b/support-files/mariadb.service.in
@@ -26,8 +26,6 @@ After=network.target
 
 [Install]
 WantedBy=multi-user.target
-Alias=mysql.service
-Alias=mysqld.service
 
 
 [Service]
